next.config.mjs

```diff
@@ -1,4 +1,6 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  distDir: "dist",
+};
```

Comment:
Let's add this too, so we can export the app to IPFS and fix webpack lib issues.

```suggestion
const nextConfig = {
  distDir: "dist",
  output: "export",
  webpack(config) {
    config.resolve.fallback = { "pino-pretty": false, encoding: false };

    return config;
  },
};
```